### DISCUSSION TITLE
Placeholder text in near invisible in dark mode

### DISCUSSION THREAD

**Discussion by robacarp** [AllYourBot/hostedgpt#221] [General]
I’m not sure if this is app wide or just on the Assistant settings dialog, but this text is hard to see:

![IMG_6706](https://github.com/allyourbot/hostedgpt/assets/208647/7c4057fe-01bf-48f2-b3af-aca8bc2d4ff3)

**Comment by keithschacht** [comment]
@robacarp I definitely fixed this in the last few days. I think your deploy may be behind.

**Comment by keithschacht** [comment]
But I definitely appreciate all these bug reports! I’m excited you’re trying things out.

**Comment by robacarp** [comment]
Dang yep sorry.
